config/routes.rb

```diff
@@ -108,7 +108,7 @@
   resolve('GreenLanes::CategoryAssessmentSearch') { [:category_assessments] }
 
   scope constraints: ->(_req) { TradeTariffFrontend::ServiceChooser.uk? } do
-    namespace :green_lanes do
+    namespace :green_lanes, path: 'check-spimm-eligibility' do
```

Comment:
So 100% of this app uses underscores. Can we just use underscores despite gds guidance?

Reply:
I've checked, and they still want dashes because they are easier to type and aligned with all the Gov UK websites.

Reply:
Let's just stick with being consistent with this application and we can review our paths for the entire application at a later date

Reply:
Ok, at the end we've agreed on _ for now